src/main/java/numble/carrot/domain/user/service/UserService.java

```diff
@@ -0,0 +1,49 @@
+package numble.carrot.domain.user.service;
+
+import lombok.RequiredArgsConstructor;
+import numble.carrot.domain.user.dto.User;
+import numble.carrot.domain.user.dto.UserJoinForm;
+import numble.carrot.domain.user.dto.UserLoginForm;
+import numble.carrot.domain.user.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public Long join(UserJoinForm userJoinForm) {
+        if(userRepository.findByEmail(userJoinForm.getEmail()).isPresent()) {
+            throw new RuntimeException("이미 존재하는 유저 입니다.");
+        }
+        User user = userJoinForm.toEntity();
+        return userRepository.save(user).getId();
+    }
+
+    public User findMember(String email) {
+        return userRepository.findByEmail(email).orElseThrow(()->{
+            try {
+                throw new ClassNotFoundException();
+            } catch (ClassNotFoundException e) {
+                e.printStackTrace();
+            }
+            return null;
+        });
+    }
+
+    @Transactional
+    public boolean login(UserLoginForm userLoginForm) {
+        Optional<User> checkEmail = userRepository.findByEmail(userLoginForm.getEmail());
+        if (checkEmail.isPresent()) {
+            if (checkEmail.get().getPassword().equals(userLoginForm.getPassword())) {
+                return true;
+            }
```

Comment:
```suggestion
if (checkEmail.isPresent()) {
            return checkEmail.get().getPassword().equals(userLoginForm.getPassword());
        }
```

이렇게 simply하게 변경할 수 있을 것 같아요!